### DISCUSSION TITLE
Is this project no longer being updated?

### DISCUSSION THREAD

**Discussion by MartinWillem** [bttv-android/bttv#664] [Q&A]
I really like having the bttv/7tv functionalities on android, but unfortunately this app has a lot of bugs like emotes being split in half, messages being cut off, the chat box not expanding when selecting so what you are writing is unreadable if it's more than 2 words and the official twitch app has been updated several times since the last release of this project, and as such several functionalities are now not yet introduced into the bttv mod app, for example alerts and watch streaks.

**Comment by FoseFx** [comment]
https://github.com/bttv-android/bttv/discussions/669#discussioncomment-10232619
